source/includes/common-minio-tiering.rst

```diff
@@ -81,4 +81,52 @@ aliases have the necessary permissions for creating policies and users
 on the deployment. See :ref:`minio-users` and :ref:`MinIO Policy Based Access Control <minio-policy>` for more
 complete documentation on MinIO users and policies respectively.
 
-.. end-create-transition-user-desc
\ No newline at end of file
+.. end-create-transition-user-desc
+
+.. start-transition-metadata-desc
+
+MinIO creates metadata for each transitioned object that identifies its
+location on the remote storage. This metadata is *required* for accessing the
+object. Applications cannot directly access the remote backend and distinguish
+a transitioned object independent of MinIO. Modification or loss of a 
+transitioned object's metadata file can result in the remote data becoming
+"disconnected" from the source. 
+
+.. end-transition-metadata-desc
+
+.. start-transition-bucket-access-desc
+
+MinIO *requires* exclusive access to the data on the remote storage tier.
+Automatic transition and transparent object retrieval depend on
+the following assumptions:
+
+- No external mutation, migration, or deletion of objects on the remote storage. 
+- No lifecycle management rules (e.g. transition or expiration) on the remote 
+  storage bucket.
+
+MinIO strongly recommends specifying a prefix on the remote bucket to better
```

Comment:
We no longer require the user to select a unique prefix in the remote bucket. MinIO server prefixes `deployment-id/source-bucket` to remote objects.

Reply:
So my thought here is the *user* has no idea how to map deployment-id back to the deployment. So if they have a complete site failure and need to purge the remote backend, they wont know that *this* ID is associated to the downed site.

If we have a way to map deployment id back to the deployment I can document that instead

Reply:
I agree that having a prefix associated with a deployment can help the user especially at the time of disaster recovery.
To clarify, this recommendation could be useful for operations, not required by MinIO.